benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  len: [64, 256, 1024, 4096, 32768],
+  dur: [5]
+});
+
+const exec = require('child_process').exec;
+function main(conf) {
+  bench.start();
+
+  const dur = +conf.dur;
+  const len = +conf.len;
+
+  const msg = '"' + Array(len).join('.') + '"';
+  const options = {'stdio': ['ignore', 'pipe', 'ignore']};
+  // NOTE: Command below assumes bash shell.
+  const child = exec(`while\n  echo ${msg}\ndo :; done\n`, options);
```

Comment:
Is it necessary that `exec()` be used for this? What about using `spawn()` with the `shell` option? That way node isn't buffering data unnecessarily.